src/Components/WishList.js

```diff
@@ -0,0 +1,36 @@
+import "../styles.css";
+import WishItem from "./wishItem";
+import { useCart } from "../cart-context";
+
+export default function WishList() {
+  const { items, setItems } = useCart();
+  // console.log(items);
+  let length = items.reduce((len, item) => (item.wishlist ? len + 1 : len), 0);
+
+  const moveToCart = (id) => {
+    let temp = items.map((el) =>
+      el.id === id ? { ...el, count: el.count + 1, wishlist: !el.wishlist } : el
```

Comment:
You can use a reducer ,to maintain different states like appending,removing,changing state